tmp/build/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
nit: weird to include a dir called `tmp/`, maybe separate the build scripts into a folder separate from the target output directory?

Reply:
Agreed and removed.